lib/dish.rb

```diff
@@ -0,0 +1,12 @@
+class Dish
+  attr_reader :price
+
+  def initialize(name:, price:)
+    @name = name
+    @price = price
+  end
+
+  def to_s
+    "#{@name} - £#{format("%<num>0.2f", num: @price)}"
+  end
+end
```

Comment:
Great idea to create a separate Dish class with prices - it makes it much easier to access price to calculate order total later on.